Add Nois Network Mainnet - Brings fair randomness to the interchain

kaisbaccour opened the pull request.

webmaster128 commented on cosmos/nois.json:
```diff
@@ -0,0 +1,43 @@
+{
+  "rpc": "https://rpc-nois.sr20de.xyz/",
+  "rest": "https://api.noismain.mcbnode.online",
```
Should we add the proxy endpoints here?

```suggestion
  "rpc": "https://rpc.cosmos.directory/nois",
  "rest": "https://rest.cosmos.directory/nois",
```

HeesungB commented:
@kaisbaccour 
Failed to get response /status from rpc endpoint.
Could you check your rpc endpoint?

kaisbaccour commented:
> @kaisbaccour Failed to get response /status from rpc endpoint. Could you check your rpc endpoint?
@HeesungB 
I just switched to polkachu (another rpc endpoint). Should be better now.

kaisbaccour commented:
Hey @HeesungB Any idea how we could solve this ? Thanks a lot

HeesungB commented:
@kaisbaccour 
If you use this for `https://nois-rpc.nysa.network`, it works.
Do you guys have a node up and running?
You must enter `node provider information` and change the image size to 256x256.

kaisbaccour commented:
@HeesungB We don't run as Nois devs nodes that are up all times.However most validators do.
I made a commit to 
- change the rpc to `https://nois-rpc.nysa.network`.
- added the `node provider info`
- changed the image size to 256x256

HeesungB commented:
@kaisbaccour @webmaster128 Your PR merged.
Thank you for contribution!
